Translation corrections (#455)
07th-mod/higurashi-console-arcs: Update/ura_016.txt

//r銃を奪い、人質を取って篭城し、警官たちを殺した凶悪犯。…その家族として、暮らしづらい日常はさらに肩身が狭くなり、苦しかった生活はさらに困窮した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "銃を奪い、人質を取って篭城し、警官たちを殺した凶悪犯。…その家族として、暮らしづらい日常はさらに肩身が狭くなり、苦しかった生活はさらに困窮した。",

> 			NULL, "These thugs had robbed a gun shop, took hostages, and murdered police officers. ...So as the brother of one of them, his painful daily life was now getting more and more difficult and shameful.", Line_Normal);

Reviewer: ```suggestion
			NULL, "These thugs had robbed a gun shop, took hostages, and murdered police officers. ...So as the brother of one of them, his painful daily life was now getting more and more shameful and difficult.", Line_Normal);
```

			NULL, " If your father... hadn't killed my brother!", Line_WaitForInput);
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1192", 256, TRUE);
	OutputLine(NULL, "　社会の連中が犯罪者の家族に、もっと目を向ける温容があれば！",
			NULL, " If society had been sympathetic to me as the brother of a criminal!", Line_WaitForInput);
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1193", 256, TRUE);
	OutputLine(NULL, "　こんなことにはならなかった！",

> 			NULL, " None of this would come to pass!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " None of this would have come to pass!", Line_WaitForInput);
```

			NULL, "\"...Yes, I am.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe3195", 256, TRUE);
	OutputLine(NULL, "少なくとも逃げ続けてきたあんたの言うことなんて、一顧にも値しない。",

> 			NULL, " At the very least, the words of a man who does nothing but run aren't even worth considering.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " At the very least, the words of a man who does nothing but run away aren't even worth considering.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe3198", 256, TRUE);
	OutputLine(NULL, "「助けを求める前に、相手が手を差し伸べてもいいと思うだけの努力もせず、",

> 			NULL, "\"You're the one who didn't make any effort to even think someone might be willing to reach out to you before seeking help. ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"You're the one who didn't make any effort to even consider that someone might be willing to reach out to you before seeking help. ", GetGlobalFlag(GLinemodeSp));
```
